tests/qchem/of_tests/test_convert.py

```diff
@@ -950,6 +1012,11 @@ def test_import_state_error():
     with pytest.raises(ValueError, match="The supported objects"):
         _ = qchem.convert.import_state(myci)
 
+    mytuple = (np.array([[3, 0], [0, 3]]), np.array([0]))
+
+    with pytest.raises(ValueError, match="For tuple input"):
+        _ = qchem.convert.import_state(mytuple)
```

Comment:
Do we need `_ = `? Can we call only `qchem.convert.import_state(mytuple)`?

Reply:
Thank you for flagging this -- removed this!